packages/ai/core/speak/speak-result.ts

```diff
@@ -0,0 +1,34 @@
+import { JSONValue } from '@ai-sdk/provider';
+import { SpeechModelResponseMetadata } from '../types/speech-model-response-metadata';
+import { SpeechWarning } from '../types';
+
+/**
+The result of a `speak` call.
+It contains the audio data and additional information.
+ */
+export interface SpeechResult {
+  /**
+   * The audio data as a binary buffer.
+   */
+  readonly audioData: ArrayBuffer;
+
+  /**
+   * The content type of the audio data (e.g., 'audio/mp3', 'audio/wav').
+   */
+  readonly contentType: string;
+
+  /**
+  Warnings for the call, e.g. unsupported settings.
+     */
+  readonly warnings: Array<SpeechWarning>;
+
+  /**
+  Response metadata from the provider. There may be multiple responses if we made multiple calls to the model.
+   */
+  readonly responses: Array<SpeechModelResponseMetadata>;
```

Comment:
single response in this case? or is there a reason it can be multiple?

Reply:
Kept it as an array to make it more inline with all the other responses (incl. transcribe) but happy to flatten if you think it's better

Reply:
lets leave, might come in handy in the future